app/views/partials/existing_group_paginated_selectable_team_members_table_form.scala.html

```diff
@@ -96,14 +96,9 @@
             @msgs("select.members.list.h1")
             </legend>
 
-            <div id="teamMembers" tabindex="0">
+            <div id="members" tabindex="0">
                 <div id="select-all"></div>
 
-            @resultsSummary.map { summary =>
```

Comment:
think you forgot about the tests that expect this summary.... were we actually meant to remove it or just move it up the page? I forget